### PR TITLE
Form sliders

### PR DESCRIPTION
**Pull request by shrutiyer**

### CODE REVIEW

File: public/javascripts/d3Controller.js [hunk 1]
```diff
@@ -3,29 +3,35 @@ var d3Controller = function($scope, $http, $state) {
...
+      return newList
+    }
+    var attendanceList = [];
+    $scope.attendanceData = formatPieData($scope.attendance, attendanceList)
+
+    var warningsList = [];
+    $scope.warningData = formatPieData($scope.warnings, warningsList)
 
-    var attendenceCount = {};
     //creating a dictionary mapping elements to their counts in a list
```
Comment (davidabrahams) [inline public/javascripts/d3Controller.js]: are all these comments necessary?

File: public/javascripts/studentControllers.js [hunk 2]
```diff
@@ -26,6 +26,9 @@ var studentController = function($scope,  $http, $state) {
 };
 
 var addEntryController = function($scope, $http, $location) {
+  // $scope.minSlider = {
```
Comment (davidabrahams) [inline public/javascripts/studentControllers.js:29]: remove?

File: public/views/content/d3.html [hunk 3]
```diff
@@ -1,3 +1,18 @@
-<nvd3 options="pieOptions" data="attendanceData"></nvd3>
+<div class="viz-form">
 
-<nvd3 options="histOptions" data="histData"></nvd3>
\ No newline at end of file
+	<div class="form-group col-md-4">
```
Comment (davidabrahams) [inline public/views/content/d3.html]: fix indentation

File: public/views/content/entryForm.html [hunk 4]
```diff
@@ -90,6 +90,14 @@
         <label>Teacher Feedback:</label>
         <input type="text" ng-model="newEntry.teacherFeedback" placeholder="Input any teach feedback." class="form-control" />
       </div>
+
```
Comment (davidabrahams) [inline public/views/content/entryForm.html:93]: remove

File: public/javascripts/d3Controller.js [hunk 5]
```diff
@@ -41,8 +47,8 @@ var d3Controller = function($scope, $http, $state) {
         legend: { margin: { top: 5, right: 35, bottom: 5, left: 0 } }
       }
     };
-    //setting attendance data for plotting to angular var
-    $scope.attendanceData = attendanceDataList;
+    // //setting attendance data for plotting to angular var
```
Comment (davidabrahams) [inline public/javascripts/d3Controller.js:50]: double comment

**Comment by davidabrahams** [comment]
:+1:

**Comment by aconsilvio** [comment]
👍

**Comment by BrennaManning** [comment]
:+1:

**Pull request merged**